main.tf

```diff
@@ -1,5 +1,11 @@
+module "label" {
```

Comment:
we don't need this label, `context.tf` has a label already which can be used if needed.
We introduce another label into a module if we need to add additional attributes or tags.